cores/arduino/stm32/stm32_def_build.h

```diff
@@ -414,6 +414,14 @@
     #define CMSIS_STARTUP_FILE "startup_stm32u575xx.s"
   #elif defined(STM32U585xx)
     #define CMSIS_STARTUP_FILE "startup_stm32u585xx.s"
+  #elif defined(STM32U595xx)
+    #define CMSIS_STARTUP_FILE "startup_stm32u595xx.s"
+  #elif defined(STM32U599xx)
+    #define CMSIS_STARTUP_FILE "startup_stm32u599xx.s"
+  #elif defined(STM32U5A5xx)
+    #define CMSIS_STARTUP_FILE "startup_stm32u5a5xx.s"
+  #elif defined(STM32U5A9xx)
+    #define CMSIS_STARTUP_FILE "startup_stm32u5a9xx.s"
```

Comment:
Like there is new mcu. The variant for U5 should be regenerated to take then in count.

Reply:
Done